esmvaltool/diag_scripts/zmnam/zmnam_preproc.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+# Note: requires system install of cdo
```

Comment:
Would it be possible to use esmvaltool preprocessor functions for this?

If not, please add cdo as a requirement in environment.yml and meta.yml so it gets installed.

Reply:
As of now it seems that it is not possible. I will add it to the requirements, thanks

Reply:
Added `cdo` to:
- dependencies list in `environment.yml`
- requirements/run in `meta.yml`

Reply:
The latest version 1.9.5 of cdo gives problem due to missing libhdf5;  working with 1.9.3